### PR TITLE
Index Settings Contributor sample

### PR DESCRIPTION
**Pull request by luanmaoski**

### CODE REVIEW

**Comment by luanmaoski** [comment]
Hi @gamerson, these two commits [here](https://github.com/gamerson/liferay-blade-samples/pull/497/commits/474d056dabe58ad4a17be01d592f811d311bb8ca) and [here](https://github.com/gamerson/liferay-blade-samples/pull/497/commits/eab3531ae8930fe7b98ef4e647fa8530a3131b19) had to be done to make the `gradlew checkSourceFormatting assemble` pass.

When i run this command on my sample it works fine.

**Comment by gamerson** [comment]
ci:test:71

**Comment by luanmaoski** [comment]
ci:test:71

**Comment by luanmaoski** [comment]
ci:test:71

**Comment by luanmaoski** [comment]
Hi @gamerson, when i ran `gradlew testIntegration` inside "liferay-workspace" i got this error `org.jboss.arquillian.container.spi.client.container.LifecycleException: Error connecting to Karaf MBeanServer: `

Since this error isn't related to my sample, could you merge it?

Thanks.

**Comment by gamerson** [comment]
yes @luanmaoski those errors are form the functional test and we don't have that for your sample yet, so I'll merge.

File: liferay-workspace/apps/service-builder/jdbc/jdbc-service/build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ dependencies {
 
 	testCompile group: "biz.aQute.bnd", name: "biz.aQute.remote.api", version: "3.3.0"
 	testCompile group: "com.google.guava", name: "guava", version: "23.0"
-	testCompile group: "com.liferay.arquillian", name: "com.liferay.arquillian.arquillian-container-liferay", version: "1.0.6"
+	testCompile group: "com.liferay", name: "com.liferay.arquillian.extension.junit.bridge", version: "1.0.6"
```
Comment (gamerson) [inline liferay-workspace/apps/service-builder/jdbc/jdbc-service/build.gradle]: I don't think we can make these changes, even though I know it is breaking SF.  @lawrence-lee   how did these pass SF in the past?  Perhaps we need to add a exclusion?

File: liferay-workspace/apps/service-builder/jndi/jndi-service/src/testIntegration/java/com/liferay/blade/samples/jndiservicebuilder/test/ConnectionTest.java [hunk 2]
```diff
@@ -58,7 +74,8 @@ public void testRegionService() throws Exception {
 
 	private static void _addData() throws Exception {
 		_executeNonQuery(
-			"create table region(id bigint not null primary key, name varchar(255));");
+			"create table region" +
```
Comment (gamerson) [inline liferay-workspace/apps/service-builder/jndi/jndi-service/src/testIntegration/java/com/liferay/blade/samples/jndiservicebuilder/test/ConnectionTest.java]: This looks like a dangerous change made by SF.

Comment (gamerson) [reply liferay-workspace/apps/service-builder/jndi/jndi-service/src/testIntegration/java/com/liferay/blade/samples/jndiservicebuilder/test/ConnectionTest.java]: But I guess it knows what it is doing?

File: liferay-workspace/apps/service-builder/jdbc/jdbc-service/build.gradle (hunk 1 above)
Comment (lawrence-lee) [reply liferay-workspace/apps/service-builder/jdbc/jdbc-service/build.gradle]: they mustve made SF even more strict. It used to not care about the arquillian package inside build.gradle

File: liferay-workspace/apps/service-builder/jndi/jndi-service/src/testIntegration/java/com/liferay/blade/samples/jndiservicebuilder/test/ConnectionTest.java (hunk 2 above)
Comment (lawrence-lee) [reply liferay-workspace/apps/service-builder/jndi/jndi-service/src/testIntegration/java/com/liferay/blade/samples/jndiservicebuilder/test/ConnectionTest.java]: This file shouldve been ignored for SF. Is this rebased from latest?

Comment (lawrence-lee) [reply liferay-workspace/apps/service-builder/jndi/jndi-service/src/testIntegration/java/com/liferay/blade/samples/jndiservicebuilder/test/ConnectionTest.java]: https://github.com/liferay/liferay-blade-samples/blob/7.1/liferay-workspace/apps/service-builder/jndi/jndi-service/source-formatter.properties

**Comment by luanmaoski** [comment]
Hi @gamerson, i rebased my branch with master and reverted the changes made on `jdbc-service/build.gradle` and `ConectionTest.java`.

When i ran `diff.sh` i got a error that isn't related with my sample.

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-blade-samples/pull/518. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**